dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -883,6 +915,150 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		GameMessages::SendSetName(entity->GetObjectID(), GeneralUtils::UTF8ToUTF16(name), UNASSIGNED_SYSTEM_ADDRESS);
 	}
 
+	if (chatCommand == "record-act" && entity->GetGMLevel() >= eGameMasterLevel::DEVELOPER) {
+		EntityInfo info;
+		info.lot = 2097253;
+		info.pos = entity->GetPosition();
+		info.rot = entity->GetRotation();
+		info.scale = 1;
+		info.spawner = nullptr;
+		info.spawnerID = entity->GetObjectID();
+		info.spawnerNodeID = 0;
+		info.settings = {
+			new LDFData<std::vector<std::u16string>>(u"syncLDF", { u"custom_script_client" }),
+			new LDFData<std::u16string>(u"custom_script_client", u"scripts\\ai\\SPEC\\MISSION_MINIGAME_CLIENT.lua")
+		};
+
+		// If there is an argument, set the lot
+		if (args.size() > 0) {
+			if (!GeneralUtils::TryParse(args[0], info.lot)) {
+				ChatPackets::SendSystemMessage(sysAddr, u"Invalid lot.");
+				return;
+			}
+		}
+
+		// Spawn it
+		auto* actor = Game::entityManager->CreateEntity(info);
+
+		// If there is an argument, set the actors name
+		if (args.size() > 1) {
+			actor->SetVar(u"npcName", args[1]);
+		}
+
+		// Construct it
+		Game::entityManager->ConstructEntity(actor);
+
+		auto* record = Cinema::Recording::Recorder::GetRecorder(entity->GetObjectID());
+
+		if (record) {
+			record->Act(actor);
+		} else {
+			Game::logger->Log("SlashCommandHandler", "Failed to get recorder for objectID: %llu", entity->GetObjectID());
```

Comment:
LOG macro should be used here, applies to all other non macro usages so ill just leave thisone